cse-adapter-app/src/main/java/com/farao_community/farao/cse/adapter/app/CseAdapterListener.java

```diff
@@ -43,34 +39,22 @@ public CseAdapterListener(CseClient cseClient, CseAdapterConfiguration cseAdapte
     }
 
     @Bean
-    public Consumer<Flux<Message<TaskDto>>> handleRun() {
+    public Consumer<Flux<TaskDto>> handleRun() {
         return f -> f
             .onErrorContinue((t, r) -> LOGGER.error(t.getMessage(), t))
-            .subscribe(this::runRequestFromMessage);
+            .subscribe(this::runRequest);
     }
 
-    void runRequestFromMessage(Message<TaskDto> message) {
-        Channel channel = Objects.requireNonNull(message.getHeaders().get(AmqpHeaders.CHANNEL, Channel.class));
-        Long deliveryTag = Objects.requireNonNull(message.getHeaders().get(AmqpHeaders.DELIVERY_TAG, Long.class));
-        try {
-            channel.basicAck(deliveryTag, false);
-        } catch (IOException e) {
-            LOGGER.error("Impossible to ack CSE run message", e);
-            throw new CseAdapterException("Impossible to ack CSE run message");
-        }
-        runRequestFromTaskDto(Objects.requireNonNull(message.getPayload()));
-    }
-
-    void runRequestFromTaskDto(TaskDto taskDto) {
+    void runRequest(TaskDto taskDto) {
```

Comment:
Not related to this changes, but i will give more context to log below or merge it with logs under case statements (example Sending IDCC request for TS : ts).

Reply:
done